### DISCUSSION TITLE
How to select and copy any text?

### DISCUSSION THREAD

**Discussion by LoveYourMommy** [directvt/vtm#178] [Q&A] [answered]
How to select and copy any text?

**Comment by o-sdn-o** [comment, marked as answer]
At the moment there are two ways to select and copy text:

- Using `Ctrl + RightMouseDrag` or `Ctrl + MiddleMouseDrag` - the selected rectangular area will be copied with all graphical attributes (SGR) to the clipboard. It requires `OSC 52` support from your terminal.
- Using `Shift + LeftDrag` - select and copy using the functionality of your terminal emulator. Some terminals may have a different selection/copy configuration than this.

Copying the text inside the built-in vtm terminal is ~not done yet~. You can track it here #149.
